homeassistant/components/deconz/sensor.py

```diff
@@ -108,6 +110,15 @@ class DeconzSensorDescription(Generic[T], SensorEntityDescription):
 
 
 ENTITY_DESCRIPTIONS: tuple[DeconzSensorDescription, ...] = (
+    DeconzSensorDescription[AirPurifier](
+        key="air_purifier_filter_run_time",
+        supported_fn=lambda device: True,
+        update_key="filterruntime",
+        name_suffix="Filter Time",
+        value_fn=lambda device: device.filter_run_time,
```

Comment:
How accurate is the run time? Is it changing a lot? If so, consider using [`ignore_variance`](https://github.com/home-assistant/core/blob/dev/homeassistant/components/prusalink/sensor.py#L169-L172)

Reply:
Good question. Im not sure. Will look at your suggestion

Reply:
I've pinged users with the device for this information. Maybe its ok to merge this as is and I will put up a future PR to tweak this based on feedback?

Reply:
Looking at logs provided in https://github.com/Kane610/deconz/issues/322 it does not update very often. The logs span 15 minutes and the value has not changed during that period